src/inflators/image-loader.ts

```diff
@@ -1,10 +1,13 @@
 import { HubsWorld } from "../app";
 import { ProjectionMode } from "../utils/projection-mode";
 import { inflateMediaLoader } from "./media-loader";
+import { MediaImageLoaderData } from "../bit-components";
 
 export interface ImageLoaderParams {
   src: string;
-  projection: ProjectionMode;
+  projection: string;
+  alphaMode: string;
```

Comment:
Nit: Maybe better to use `PROJECTION_MODE` and `ALPHA_MODE` types here? Probably you use strings type here because this inflator is (currently) only for glTF and no guarantee that valid data comes from glTF. But expected type will be clearer if we use `PROJECTION_MODE` and `ALPHA_MODE`. And it would be safer even if we end up using this inflator for jsx in the future.

SImilarly, in `video-loader.ts`.